src/coreclr/debug/daccess/request.cpp

```diff
@@ -4727,11 +4732,15 @@ HRESULT ClrDataAccess::GetReJITInformation(CLRDATA_ADDRESS methodDesc, int rejit
     SOSDacLeave();
 
     return hr;
+#else
+    return E_NOTIMPL;
```

Comment:
Should this just return `pReJitData->flags = DacpReJitData2::kActive;` - same as with no code versioning?

Reply:
Yes, more functional approach.